networkx/generators/trees.py

```diff
@@ -44,55 +58,44 @@ def prefix_tree(paths):
         of the root node is None, and the 'source' of the nil node is
         :data:`.NIL`.
```

Comment:
The node is `-1` but the "source" for that node is "NIL".  I made some changes to make this more clear. And I took out the `:data:` link since it should not link to anything now.  Thanks!